app/src/main/res/values/dimens.xml

```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <!-- Default screen margins, per the Android Design guidelines. -->
+  <dimen name="activity_horizontal_margin">16dp</dimen>
+  <dimen name="activity_vertical_margin">16dp</dimen>
+  <dimen name="nav_header_vertical_spacing">8dp</dimen>
+  <dimen name="nav_header_height">176dp</dimen>
+  <dimen name="margin_30">30dp</dimen>
+  <dimen name="image_dim_24">24dp</dimen>
+  <dimen name="margin_10">10dp</dimen>
```

Comment:
The margin/padding name ending with a fix value means that it won't change, and the values which do not change at all it is preferable not to declare them in `dimens.xml` file.
Example: `activity_horizontal_margin ` is fine to use but `margin_30` is not.

Reply:
activity_horizontal_margin is already there

Reply:
Removed